examples/animation/animated_ui.rs

```diff
@@ -40,7 +40,7 @@ fn main() {
 }
 
 impl AnimatableProperty for FontSizeProperty {
-    type Component = TextStyle;
+    type Component = TextFont;
```

Comment:
If I understand the bevy error 0005 correctly, we shouldn't generally be changing the size of a font this way, and instead use transform or ui_scale. Should this example continue to teach this possible footgun to the users?

Reply:
Yeah I agree it's not a good example, it should probably be removed. Scaling the foot using transform isn't normally a good solution either as the text will end up blurred (or blocky if you turn off anti-aliasing). It's outside the scope of this PR to deal with it though I think.